tools/cudaq-qpud/nvqc_proxy.py

```diff
@@ -194,24 +192,26 @@ def do_POST(self):
                     save_dir = os.getcwd()
                     os.chdir(pathlib.Path(temp_file.name).parent)
                     cmd_list = build_command_list(temp_file.name)
-                    cmd_result = subprocess.run(cmd_list,
-                                                capture_output=False,
-                                                text=True)
-                    if cmd_result.returncode == 0:
+                    try:
+                        subprocess.run(cmd_list,
+                                       timeout=WATCHDOG_TIMEOUT_SEC,
```

Comment:
If we need any `WATCHDOG_TIMEOUT_SEC` changes, can we please save them for a different PR? Unless I'm missing something, I don't think any of those changes are required for this PR, and I'm concerned that this change doesn't work when `RUN_AS_NOBODY` is set, like it will be on NVQC.